### DISCUSSION TITLE
SunBangla India

### DISCUSSION THREAD

**Discussion by debctg** [iptv-org/community#768] [Q&A]
### Channel Name

Sun Bangla

### Channel ID

SunBangla.in

### Notes

I am looking Indian Sun Bangla m3u playlist please.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19759
